WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostPreviewFragment.java

```diff
@@ -16,11 +16,16 @@
 import org.wordpress.android.WordPress;
 import org.wordpress.android.fluxc.model.PostModel;
 import org.wordpress.android.fluxc.model.SiteModel;
+import org.wordpress.android.fluxc.store.PostStore;
 import org.wordpress.android.util.StringUtils;
 import org.wordpress.android.util.ToastUtils;
 import org.wordpress.android.util.WPHtml;
 
+import javax.inject.Inject;
+
 public class EditPostPreviewFragment extends Fragment {
+    @Inject PostStore mPostStore;
```

Comment:
Going to need to inject this fragment into the `Component`, otherwise `PostStore` will always be `null`.